### PR TITLE
New logic for error handling in the frontend

### PR DESCRIPTION
**Pull request by vin-ni**
- Externalized the Error frame to dynamically show it
    - It's waiting 3 seconds to debounce the graphql connection & mounting of the stream
- Added spinner until stream is mounted
- Fixed fade in of the intro card
- General styling of the error div

### CODE REVIEW

File: caster-front/src/components/GraphDetailTemplates/DrifterTemplate.vue [hunk 1]
```diff
@@ -99,47 +100,81 @@ const shiftDialogs = () => {
...
+  if (waitingTimeout.value) {
+      if(!data.value || data.value?.streamInfo.__typename === 'NoStreamAvailable' || data.value.streamInfo.stream.streamPoint === undefined ) {
+        return true;
+      } else {
+        return false;
+      }
+  } else {
+    return false;
+  }
+});
```
Comment (capital-G) [inline caster-front/src/components/GraphDetailTemplates/DrifterTemplate.vue:126]: Could be written with less nesting like this

```typescript
    if(waitingTimeout.value && (
        !data.value ||
        data.value?.streamInfo.__typename === 'NoStreamAvailable' ||
        data.value.streamInfo.stream.streamPoint === undefined
    )) {
      return true;
    }
    return false;
```

**Review by capital-G — COMMENTED** [review]
I also tried to fix this but to me this looks like a bug in element-plus loading mechanism, see https://github.com/element-plus/element-plus/issues/13285

File: caster-front/src/components/PlayerComponent.vue [hunk 2]
```diff
@@ -271,6 +272,7 @@ onMounted(async () => {
   await initJanus();
   // todo this is rather nasty way of setting things up
   await setTimeout(() => {
+    playerMounted.value = true;
```
[the remark this answers is not in the document]
Comment (vin-ni) [reply caster-front/src/components/PlayerComponent.vue:275]: We're doing this above at line 251.

[the remark this answers is not in the document]
Comment (capital-G) [reply caster-front/src/components/PlayerComponent.vue:275]: you are right :)

**Pull request merged**